src/components/common/view/AddWebApp.web.js

```diff
@@ -105,7 +105,7 @@ const ExplanationDialog = withStyles(mapStylesToProps)(({ styles }) => {
 const AddWebApp = props => {
   const store = SimpleStore.useStore()
   const [showDialog] = useDialog()
-  const { show, showPopupFromMenu } = store.get('addWebApp')
+  const { show, showMenuItem } = store.get('addWebApp')
```

Comment:
still needs better name

Reply:
done in https://github.com/GoodDollar/GoodDAPP/pull/1144/commits/285d59a8a7193d36edf2f29994c30adf855968af